tensorflow/python/ops/image_ops_impl.py

```diff
@@ -1601,6 +1623,12 @@ def random_contrast(image, lower, upper, seed=None):
   Returns:
     The contrast-adjusted image(s).
 
+  Usage Example:
+    ```python
+    import tensorflow as tf
+    x = tf.random.normal(shape=(256, 256, 3))
+    tf.image.random_contrast(x, 0, 10)
+    ```
```

Comment:
Let's maintain uniformity and leave a line after the comment as you have done with the previous examples

Reply:
Just changed it. I'll make sure to keep that in mind in the future! Thanks for the feedback :)